packages/create-skill-app/templates/next/.config-templates/env.development

```diff
@@ -32,5 +32,5 @@ NEXT_PUBLIC_CONVERTKIT_GET_SUBSCRIBER_URL=/api/skill/subscriber/convertkit
 
 # Sanity
 NEXT_PUBLIC_SANITY_API_VERSION=2023-01-03
-NEXT_PUBLIC_SANITY_PROJECT_ID=project_id_from_sanity
+NEXT_PUBLIC_SANITY_PROJECT_ID=fr5ycba3
```

Comment:
I intentionally left this defined pointing at sanity starter so that new products are not completely blank.